src/wizard/wizard.scss

```diff
@@ -27,68 +26,98 @@
+      border: 1px solid t(iui-color-foreground-positive);
+      background-color: t(iui-color-background-1);
+      color: t(iui-color-foreground-positive);
+    }
+  }
+
+  // Current step
   &.iui-current {
+    font-weight: $iui-font-weight-semibold;
+    
     .iui-wizard-step-name {
       @include themed {
         color: t(iui-color-foreground-positive);
       }
-      font-weight: $iui-font-weight-semibold;
     }
 
     .iui-wizard-circle {
-      font-weight: $iui-font-weight-semibold;
       @include themed {
         color: t(iui-color-foreground-accessory);
         border-color: t(iui-color-background-positive);
         background-color: t(iui-color-background-positive);
       }
     }
 
-    .iui-wizard-track-content::before {
+    .iui-wizard-track-content::after {
       @include themed {
-        background-color: t(iui-color-foreground-positive);
+        background-color: t(iui-color-background-5);
       }
     }
   }
 
-  &.iui-completed {
-    &.iui-clickable .iui-wizard-circle {
-      @include iui-focus(iui-focus-positive-box-shadow);
-
-      &:hover {
-        cursor: pointer;
-        @include themed {
-          background-color: t(iui-color-background-positive);
-          border-color: t(iui-color-background-positive);
-          color: t(iui-color-foreground-accessory);
-        }
+  // Future steps
+  &.iui-current ~ .iui-wizard-step {
+    .iui-wizard-circle {
+      @include themed {
+        border: 1px solid t(iui-color-background-5);
+        background-color: t(iui-color-background-1);
+        color: t(iui-text-color-muted);
       }
     }
 
     .iui-wizard-step-name {
       @include themed {
-        color: t(iui-color-foreground-positive);
+        color: t(iui-text-color-muted);
       }
     }
 
-    .iui-wizard-circle {
+    .iui-wizard-track-content::before,
+    .iui-wizard-track-content::after {
       @include themed {
-        color: t(iui-color-foreground-positive);
-        border-color: t(iui-color-foreground-positive);
+        background-color: t(iui-color-background-5);
       }
     }
+  }
 
-    .iui-wizard-track-content::before,
-    .iui-wizard-track-content::after {
-      @include themed {
-        background-color: t(iui-color-foreground-positive);
+  // Clickable steps
+  &.iui-clickable {
+    .iui-wizard-circle {
+      @include iui-focus(iui-focus-positive-box-shadow);
+      cursor: pointer;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease-out;
+      }
+
+      &:hover {
+        @include themed {
+          color: t(iui-color-foreground-accessory);
+          border: 1px solid t(iui-color-background-positive);
+          background-color: t(iui-color-background-positive);
+        }
       }
     }
   }
 
   &:first-child .iui-wizard-track-content::before,
   &:last-child .iui-wizard-track-content::after {
-    background-color: transparent;
+    background-color: transparent !important;
```

Comment:
Do you intend to keep this? 😄

Reply:
No, this was just temporary until I figured out how to fix the issue.  It's been fixed in the latest push.